sfa_dash/templates/head.html

```diff
@@ -37,6 +37,8 @@
 <script src="/static/js/table-search.js"></script>
 <script src="/static/js/table-checkall.js"></script>
 
+<script src="https://cdn.plot.ly/plotly-latest.min.js"></script>
```

Comment:
latest? seems safer to pin. only a matter of time before COVID-19 finds its way into the filthy javascript ecosystem.

Reply:
ahh yep, thanks for that! I added that in to start testing before plotly version was available.

Reply:
Wash your imports!

Reply:
And unfortunately Plotly doesn't have the same versions across languages, so we're going to have to keep an eye on plotly :[